Source/Atomic/IO/PackageFile.h

```diff
@@ -86,7 +86,20 @@ class ATOMIC_API PackageFile : public Object
     const Vector<String> GetEntryNames() const { return entries_.Keys(); }
 
     // ATOMIC BEGIN
-    
+
+    /// Return a file name in the package, for scripting use ( GetEntryNames is not bindable? )
+    const String& GetEntryName(unsigned index) const 
+    {
+        unsigned nn = 0;  // .Begin() + index syntax not supported on HashMap
+        for (HashMap<String, PackageEntry>::ConstIterator j = entries_.Begin(); j != entries_.End(); ++j)

```

Comment:
Comment changed.  The HashMap has an order-ish and the const iterating should provide valid values for each index for the current state of the PackageFile. The object could change, and this method will not guarantee an item will remain at a specific index. GetEntryNames() only gives you a pile of names without index context, so maybe index is a bad term.